include/openssl/crypto.h

```diff
@@ -399,7 +399,7 @@ int CRYPTO_memcmp(const void * in_a, const void * in_b, size_t len);
 # define OPENSSL_INIT_ADD_ALL_MACS           0x08000000L
 # define OPENSSL_INIT_NO_ADD_ALL_KDFS        0x10000000L
 # define OPENSSL_INIT_ADD_ALL_KDFS           0x20000000L
-/* FREE: 0x40000000L */
+/* OPENSSL_INIT_BROTLI                       0x40000000L */
```

Comment:
If this is really going to be declared this way you might want to put a comment above it explaining why.

Reply:
`OPENSSL_INIT_ZLIB` is declared the same way. 

P.S. Not arguing, just explaining. If you want a comment here, then a comment for zlib would be appropriate, but then again, the intent of this PR was not to fix zlib's faults!

Reply:
I guess I dont understand why it needs a flag and then doesnt use it directly..

Reply:
It's defined in cryptlib_int.h, as it's meant to only be used internally. But this is the master list of OPENSSL_INIT flags, so it needs to be "reserved" here.
But... (BIG **BUT**) it does appear that I missed the actual use of this flag... I did all the internals, but not the use of the flag in a manner similar to OPENSSL_INIT_ZLIB. I _was_ thinking I might reuse OPENSSL_INIT_ZLIB, and make it a generic OPENSSL_INIT_COMP flag, but I didn't get around to that. I will have to look to see if sharing an init flag between ZLIB and BROTLI makes sense, or if I should use the flag I defined!

Reply:
So its a reserved for internal use flag.. Even just saying that would be good.
Those flags are nearly full :). I tried one outside the 32 bit range and bad things happened on 32 bit machines..

Reply:
Apparently, I do use it (easier to search in the workspace...)

Reply:
@slontis the flag is used used here:

https://github.com/openssl/openssl/blob/fb84ea70a0cca8c6e430906500b00ebbe4463f2c/crypto/init.c#L657-L664